code-challenges.js

```diff
@@ -8,32 +8,74 @@
 // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
 
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
-
-var temp = 42
+// var temp = Math.floor(Math.random() * 400)
+// var temp = 42
 // var temp = 350
 // var temp = 212
-
+// var temp = "abc"
+// var temp = Math.floor(Math.random() * 400)
+
+// if (temp > 212){
+//     console.log(`${temp} above boiling point`);
+// } else if (temp < 212){
+//     console.log(`${temp} below boiling point`);
+// } else if (temp === 212){
+//     console.log(`${temp} is at boiling point`);
+// } else {
+//     console.log(`${temp} is not a number`);
+// }
```

Comment:
This looks great! In the future will you comment IN the code you are submitting?